src/libcrun/cgroup-systemd.c

```diff
@@ -952,7 +952,7 @@ libcrun_destroy_cgroup_systemd (struct libcrun_cgroup_status *cgroup_status,
 
   ret = destroy_cgroup_path (cgroup_status->path, mode, err);
```

Comment:
same here, we can just `return destroy_cgroup_path (cgroup_status->path, mode, err);`